cmake-modules/AddTestCMocka.cmake

```diff
@@ -1,39 +1,15 @@
+#
+# Copyright (c) 2007      Daniel Gollub <dgollub@suse.de>
+# Copyright (c) 2007-2018 Andreas Schneider <asn@cryptomilk.org>
+# Copyright (c) 2018      Anderson Toshiyuki Sasaki <ansasaki@redhat.com>
+#
+# Modifed version from https://github.com/xbmc/libssh/blob/master/cmake/Modules/AddCMockaTest.cmake
+
 find_package(cmocka CONFIG REQUIRED)
 
 enable_testing()
 include(CTest)
 
-if (CMAKE_CROSSCOMPILING)
```

Comment:
Was all this dead code?

Reply:
not dead, but not really used by us.
This code is to use wine emulator on Windows for CYGIN or MINGW.
We don't have any documentation about how work with any of these (not even know if someone has tried).
Sp far, for Windows, we only use MSVC